recipes/zoe/all/conanfile.py

```diff
@@ -1,13 +1,13 @@
 from conan import ConanFile
 from conan.errors import ConanInvalidConfiguration
 from conan.tools.microsoft import is_msvc_static_runtime
-from conan.tools.files import get, copy, apply_conandata_patches, export_conandata_patches, rmdir
+from conan.tools.files import get, copy, apply_conandata_patches, export_conandata_patches, rmdir, replace_in_file
 from conan.tools.build import check_min_cppstd
 from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
 from conan.tools.scm import Version
 import os
 
-required_conan_version = ">=1.53.0"
+required_conan_version = ">=2"
```

Comment:
```suggestion
required_conan_version = ">=2.1"
```

implements was introduced in 2.0.9